sql/updates/world/3.3.5/XXXX_XX_XX_XX_world_335.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger`='781';
```

Comment:
This PR contains impressive work, I'm embarrassed mentioning codestyle details (for later, finishing touches). Anyway:
```sql
[ ... ] `spell_trigger`=781;
```
(the `spell_trigger` value is int 8, not text)

Cosmetic detail: could you add a -- (comment) line on top for separation from previous SQL updates?